bin/create-users.sh

```diff
@@ -47,6 +47,16 @@ if [[ -z "${TEST_LAW_FIRM_B_PASSWORD}" ]]; then
     exit 1
 fi
 
+if [[ -z "${IA_SYSTEM_USERNAME}" ]]; then
+    echo "ERROR: IA_SYSTEM_USERNAME must be defined in the environment" 1>&2
```

Comment:
These new env vars should probably be added to the `Readme` so that the full list is there

Reply:
Added